service/k8sclient/k8sclient.go

```diff
@@ -1622,13 +1623,25 @@ func (c *K8sClient) GetConsumedDiskBytes(ctx context.Context, clusterType kubern
 		if err != nil {
 			return 0, errors.Wrap(err, "can't compute consumed disk size: failed to get worker nodes")
 		}
+		// TODO We want to use clientcmd.Load in the future to load the kubeconfig from slice of bytes.
```

Comment:
we do have kubeconfig at the moment we create k8sclient, so let's reuse it instead of reading from file.

Reply:
> Do we still need proxy being run?

We don't run it, do we? I removed all the code related to it, I think.

Reply:
yeah, you are right, missed that you removed proxy.go

Reply:
> we do have kubeconfig at the moment we create k8sclient, so let's reuse it instead of reading from file.

Thanks for reminding me, I got confused.